ghc-lib-gen/src/GhclibgenOpts.hs

```diff
@@ -48,7 +49,8 @@ ghclibgenVersion =
 -- | A parser of a ghc-lib-gen target: `target := | "--ghc-lib-parser"
 -- | "--ghc-lib" | /* nothing */`.
 ghclibgenTarget :: Parser GhclibgenTarget
-ghclibgenTarget = fmap (fromMaybe Ghclib) $ optional (ghclibParser <|> ghclib)
+ghclibgenTarget =
+  fromMaybe Ghclib Control.Applicative.<$> optional (ghclibParser <|> ghclib)
```

Comment:
Why did we fully qualify `<$>` here? It should already be in `Prelude`, isn't it?

Reply:
Yes. I just mechanically applied the suggestion hlint provided. I meant to come back to that in a later commit.